src/app/content/hooks/storeTextSize.ts

```diff
@@ -19,7 +19,11 @@ export const loadStoredTextSize = (services: MiddlewareAPI & AppServices) => asy
   }
 
   if (typeof window !== 'undefined' && window.localStorage) {
-    storedTextSize = window.localStorage.getItem(textResizerStorageKey);
+    try {
+      storedTextSize = window.localStorage.getItem(textResizerStorageKey);
+    } catch {
+      // They have blocked access to localStorage; ignore it
+    }
```

Comment:
trying to parse this function in my head, it looks like what its gonna do now is reset your text size back to the default every time you navigate, is that the case?

Reply:
Yes, that is what will happen.

Reply:
oh no i see now there is a short-circuit on line 17 that will noop if the state has a value, so we should be ok right? (i haven't tested, does it work or no?)

Reply:
why doesn't it load if you block all cookies?

Reply:
suppressing the error that causes the `"Uh-oh, there's been a glitch"` dialog is the major acceptance criteria for this issue

Reply:
I was able to suppress that (you can try it with the recent deployment). The book still doesn't load, though, because the failing line is [here](https://github.com/openstax/rex-web/blob/main/src/app/utils.ts#L43). I put a check to bypass `catchError` (which does the glitch dialog).

Reply:
thats just where its being re-thrown. the error is coming from the redux middleware somewhere, and then the error handler sends to sentry here https://github.com/openstax/rex-web/blob/main/src/app/utils.ts#L70, once the error is recorded in sentry it shows the "uh oh" dialog. you need to make the error not get as far as sentry.

what you're looking at here doesn't really indicate where the error is coming from, the error's stacktrace should have that, maybe if you go there you can handle the error where it happens.

alternatively you could add some more generic error handling in this error handler that you're linking to (the handling is a few lines down in the actual error handler function), but that would be for handling any instance of `SecurityError` no matter where it comes from, and we might not want to be that aggressive with suppressing the errors

Reply:
The stack trace takes the error to [here](https://github.com/openstax/rex-web/blob/main/src/app/utils.ts#L37) from [here](https://github.com/openstax/rex-web/blob/main/src/app/navigation/middleware.ts#L17). I can alter `catchError` or intercede before calling `catchError`. Either way, the dialog is prevented, but the book content doesn't get loaded.